src/domain/signup/signupServerErrorsContext/utils.ts

```diff
@@ -57,13 +84,29 @@ export const parseSignupGroupServerErrors = ({
       return [];
     }
   }
+
+  function parseContactPersonServerErrorLabel({
+    key,
+  }: {
+    key: string;
+  }): string {
+    if (isGenericServerError(key)) {
+      return '';
```

Comment:
Minor thing here too: according to SonarCloud, test coverage is missing for this line

Reply:
Added a test for generic server errors.